File: LeetCode/Easy/1313-decompress-run-length-encoded-list/1313-decompress-run-length-encoded-list.go
func decompressRLElist(nums []int) []int {
    ret := make([]int, 0)
    
    for i := 0; i < len(nums); i = i+2 {
        temp := nums[i+1]
        for j := 0; j < nums[i]; j++ { ret = append(ret, temp) }
    }
    
    return ret
}

File: LeetCode/Easy/1313-decompress-run-length-encoded-list/1313-decompress-run-length-encoded-list.cpp
class Solution {
public:
    vector<int> decompressRLElist(vector<int>& nums) {
        vector<int> ret;
        
        for (int i = 0; i < nums.size(); i = i + 2) {
            int temp = nums[i+1];
            for (int j = 0; j < nums[i]; ++j) ret.push_back(temp);
        }
        
        return ret;
    }
};